src/main/java/com/xpanxion/java/springboot/da1/demo/controller/student9/ManagerController9.java

```diff
@@ -28,14 +36,22 @@
     private JdbcTemplate jdbcTemplate;
     private final String SELECT_ALL_MANAGERS = "SELECT * FROM manager";
 
+    private final String INSERT_MANAGER = "INSERT INTO manager (full_name) VALUES (?)";
+
+
+
     public DataAccessManager() {}
 
+    public void addManager(Manager manager){
+         jdbcTemplate.update(INSERT_MANAGER,manager.getFull_name());
+    }
+
     public List<Manager> getManagerList() {
         List<Manager> managerList;
          managerList = jdbcTemplate.query(SELECT_ALL_MANAGERS, (row,rowNum) -> {
             var id = Integer.parseInt(row.getString("manager_id"));
             var full_name = row.getString("full_name");
-            return new Manager(id,full_name);
```

Comment:
We should keep this id here... probably best to make another Manager constructor if one doesn't exist.

Reply:
I went ahead and added another constructor to the Manager model so it can take a manager with and id and full name. I have added the id param to the getManagerlist